tests/fullydistributed_grids/solution_transfer_01.cc

```diff
@@ -89,6 +89,7 @@ test(TriangulationType &triangulation)
 
   {
     triangulation.load(filename);
+    dof_handler.distribute_dofs(FE_Q<dim>(2));
```

Comment:
Yes, this makes sense. The fact that this worked before is a complete coincidence and it's probably good that you broke this.